src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -1551,6 +1553,45 @@ public void unpublishFeedbackSession(String feedbackSessionName,
             throw new RuntimeException(ERROR_SENDING_EMAILS, e);
         }
     }
+    
+    public EmailWrapper sendConfirmationEmailForSubmission(String courseId, String feedbackSessionName,
+                                                           String userId, String unregisteredStudentEmail,
+                                                           String regKey)
+                    throws EntityDoesNotExistException {
+        
+        if (!isFeedbackSessionExists(feedbackSessionName, courseId)) {
+            throw new EntityDoesNotExistException(ERROR_NON_EXISTENT_FS_CONFIRM + courseId + "/" + feedbackSessionName);
+        }
+        
+        FeedbackSessionAttributes session = getFeedbackSession(feedbackSessionName, courseId);
+        StudentAttributes student = null;
+        InstructorAttributes instructor = null;
+        
+        if (userId != null) {
+            student = studentsLogic.isStudentInAnyCourse(userId)
```

Comment:
A student being in any course is not indicative of the student being in the specific course `courseId`.